instrumentation/logback/logback-appender-1.0/library/src/test/java/io/opentelemetry/instrumentation/logback/appender/v1_0/OpenTelemetryAppenderTest.java

```diff
@@ -71,10 +72,34 @@ void logNoSpan() {
     List<LogRecordData> logDataList = logRecordExporter.getFinishedLogRecordItems();
     assertThat(logDataList).hasSize(1);
     LogRecordData logData = logDataList.get(0);
-    assertThat(logData.getResource()).isEqualTo(resource);
-    assertThat(logData.getInstrumentationScopeInfo()).isEqualTo(instrumentationScopeInfo);
-    assertThat(logData.getBody().asString()).isEqualTo("log message 1");
-    assertThat(logData.getAttributes().size()).isEqualTo(4); // 4 code attributes
+
+    OpenTelemetryAssertions.assertThat(logData)
```

Comment:
I have started to introduce `OpenTelemetryAssertions` in the Logbacks tests. Perhaps the full migration could be done in another PR?